apps/src/code-studio/initApp/project.js

```diff
@@ -1129,13 +1129,29 @@ var projects = (module.exports = {
                 saveSourcesErrorCount,
                 err.message
               ).finally(() => utils.reload());
+              MetricsReporter.incrementCounter(
+                'LegacyLab.ProjectSaveFailureClient',
+                [
+                  {name: 'AppName', value: this.getStandaloneApp()},
```

Comment:
You'll need to use [this method](https://github.com/code-dot-org/code-dot-org/pull/70113/changes#diff-cb88bfae4cdc4d0faf3a6d2eca83948c249ef9f1ae0b1344e5f0a4709bb79094R911-R913) for the app name, we had an incident because `getStandaloneApp` is sometimes `null`.